pkg/odo/cli/pipelines/utility/utility.go

```diff
@@ -0,0 +1,16 @@
+package utility
+
+import (
+	"strings"
+
+	"github.com/openshift/odo/pkg/log"
+)
+
+// AddGitSuffix will append .git to URL
```

Comment:
It may not...

Maybe this is `AddGitSuffixIfNecessary` ?

Reply:
I've updated the function name